docs/out_of_core.rst

```diff
@@ -1,46 +1,28 @@
-Out of Core in Modin (experimental)
-===================================
+Out of Core in Modin
+====================
 
 If you are working with very large files or would like to exceed your memory, you may
 change the primary location of the `DataFrame`_. If you would like to exceed memory, you
-can use your disk as an overflow for the memory. This API is experimental in the context
-of Modin. Please let us know what you think!
-
-Install Modin out of core
--------------------------
-
-Modin now comes with all the dependencies for out of core functionality by default! See
-the :doc:`installation page </installation>` for more information on installing Modin.
+can use your disk as an overflow for the memory.
 
 Starting Modin with out of core enabled
 ---------------------------------------
 
-Out of core is detected from an environment variable set in bash.
-
-.. code-block:: bash
-
-   export MODIN_OUT_OF_CORE=true
-
-We also set up a way to tell Modin how much memory you'd like to use. Currently, this
-only accepts the number of bytes. This can only exceed your memory if you have enabled
-``MODIN_OUT_OF_CORE``.
+Out of core is now enabled by default for both Ray and Dask engines.
 
-[Optional]: Set a limit on the out of core space for Modin
-""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+Disabling Out of Core
+---------------------
 
-**Warning: Make sure you have enough space in your disk for however many bytes you**
-**request for your DataFrame**
+Out of core is enabled by the compute engine selected. To disable it, start your
+preferred compute engine with the appropriate arguments. For example:
 
-This limits the amount of memory that Modin can use.
-
-Here is how you set ``MODIN_MEMORY``:
-
-.. code-block:: bash
-
-  export MODIN_MEMORY=200000000000 # Set the number of bytes to 200GB
+.. code-block:: python
 
+  import modin.pandas as pd
+  import ray
 
-**The default for Modin is 8x the memory on the machine.**
+  ray.init(_plasma_directory="/tmp")  # setting to disable out of core in Ray
```

Comment:
Is there a way to disable object spilling in Dask?

Reply:
It seems you have to set a config. 

https://distributed.dask.org/en/latest/worker.html#memory-management

Reply:
Since it is said above `To disable it, start your
preferred compute engine with the appropriate arguments` we should probably provide an example both for Ray and Dask or define separate a section for Ray and Dask where we can show an example how to disable object spilling in Ray and put the link to configure object spilling in Dask if no other ways to do it.

Reply:
Ok added a link there.